docs/install/index.md

```diff
@@ -1743,4 +1744,10 @@ Will be available soon.
 </div>
 </div>
 
-# Download Source Package
+<br/>
+<!-- Download -->
+
+# Source Download
```

Comment:
@aaronmarkham the git clone section always clones the bleeding edge master. Should we move this "Source Download" to step 4 instead?